deploy-edgeX.sh

```diff
@@ -91,8 +89,6 @@ run_service device-virtual
 
 run_service app-service-configurable
 
-sleep 100s # Wait for rulesengine fully startup, because it takes around 100s on Raspberry Pi
-
```

Comment:
removed `sleep 100s # Wait for rulesengine fully startup, because it takes around 100s on Raspberry Pi`